### PR TITLE
Comment out filter paths until notebooks all run

### PR DESCRIPTION
**Pull request by bonnland**
Binderbot currently has a bug that prevents one of the notebooks from running.   We comment out the filter paths in order to have this notebook rendered and added to pangeo-gallery without running it.

### CODE REVIEW

**Review by andersy005 — APPROVED** [review]

**Comment by bonnland** [comment]
Thanks for the review, @andersy005 !

I merged a few PRs without approval because I didn't know of a different way to test the binderbot build.   In general, however, I plan to wait for approval (unless you prefer that I don't in some cases).

**Comment by andersy005** [comment]
> I merged a few PRs without approval because I didn't know of a different way to test the binderbot build. In general, however, I plan to wait for approval (unless you prefer that I don't in some cases) 

@bonnland, No worries. I personally don't have any problem with merging into the **main branch of this repo** without approval :). Also, due to how binderbot works, you may have to merge first before finding out whether changes have taken effect and having to wait for hours to get an approval may hamper your progress/productivity. I'm going to help speed up the pull requests process whenever I can.

**Pull request merged**
